### PR TITLE
Fixes urinal cakes being inedible

### PR DESCRIPTION
**Pull request by swindly**
:cl: Swindly
fix: Fixed urinal cakes being inedible
/:cl:

[why]: They were supposed to be edible but weren't

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Arent you going to add any 'gross' or flavour foodcode to your commit? This seems like a minimum effort fix.

**Comment by evsey9** [comment]
tasty

**Comment by FantasticFwoosh** [comment]
Sorry to nitpick but what does it taste like also?

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 
 /obj/structure/urinal/New()
 	..()
```
Comment (Cyberboss) [inline code/game/objects/structures/watercloset.dm]: Change this to Initialize() and make . = ..() it causes other issues by being /New()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**